src/client/common/startPage/themeFinder.ts

```diff
@@ -59,7 +59,7 @@ export class ThemeFinder implements IThemeFinder {
     public async findLanguageConfiguration(language: string): Promise<LanguageConfiguration> {
         if (language === PYTHON_LANGUAGE) {
             // Custom for python. Some of these are required by monaco.
-            return {
+            const config: any = {
```

Comment:
Does this work:

```suggestion
            const config: unknown = {
```

(`unknown` should be preferred to `any`)